Puppetfile

```diff
@@ -30,6 +30,10 @@ mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
 # Plugin modules
 mod 'puppetlabs-azure_inventory', '0.2.0'
+# TODO: Update to forge or puppetlabs namesace
+mod 'puppetlabs-terraform',
```

Comment:
```suggestion
mod 'puppetlabs-aws_inventory`,
```

Reply:
:man_facepalming: I fixed the github ref but not the module name. I did a lot of copy paste bc I did this one after the terraform one.